### PR TITLE
Make industrial_robot_simulator python3 compatible.

### PR DESCRIPTION
**Pull request by de-vri-es**
This PR makes two minor changes to make the industrial robot simulator compatible with python3. There should be no functional changes at all for python 2.

I only had to swap `operator <>` with `operator !=` and import the `Queue` as `queue` in python 3.

### CODE REVIEW

File: industrial_robot_simulator/industrial_robot_simulator [hunk 1]
```diff
@@ -34,7 +34,12 @@
 import rospy
 import copy
 import threading
-import Queue
+
+import sys
+if sys.version_info[0] == 2:
+	import Queue
+else:
+	import queue as Queue
```
Comment (gavanderhoorn) [inline industrial_robot_simulator/industrial_robot_simulator]: Could you please change indentation to spaces?

File: industrial_robot_simulator/industrial_robot_simulator [hunk 2]
```diff
@@ -34,7 +34,12 @@
 import rospy
 import copy
 import threading
-import Queue
+
+import sys
+if sys.version_info[0] == 2:
```
Comment (gavanderhoorn) [inline industrial_robot_simulator/industrial_robot_simulator]: Suggestion: use `sys.version_info.major` instead: intent is immediately clear and no magic nrs.

File: industrial_robot_simulator/industrial_robot_simulator (hunk 1 above)
Comment (de-vri-es) [inline industrial_robot_simulator/industrial_robot_simulator]: Good point, updated.

File: industrial_robot_simulator/industrial_robot_simulator (hunk 2 above)
Comment (de-vri-es) [inline industrial_robot_simulator/industrial_robot_simulator]: Agreed, changed.

**Comment by de-vri-es** [comment]
Any other thoughts?

**Comment by shaun-edwards** [comment]
+1... @de-vri-es, thanks so much for the contirbution.

**Pull request merged**